app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,283 @@
+    path_output: str
+        Path to the output directory,
+        which will be the temporary dir. of the following process
+    '''
+    with open(path_runconfig_in, 'r+', encoding='utf8') as fin:
+        runconfig_dict_in = yaml.safe_load(fin.read())
+
+    cfg = RunConfig.load_from_yaml(path_runconfig_in, 'rtc_s1')
+
+
+    list_runconfig_burst = []
+    list_logfile_burst = []
+
+    # determine the bursrs to process
+    list_burst_id = cfg.bursts.keys()
+    #runconfig_dict_in['runconfig']['groups']['product_group']['output_dir']
+
+    # determine the output directory for child process
+    time_stamp = str(float(time.time()))
+    dir_output = os.path.join(cfg.groups.product_group.scratch_path, f'temp_{time_stamp}')
+    for burst_id in list_burst_id:
+        path_temp_runconfig = os.path.join(cfg.scratch_path,
+                                           f'burst_runconfig_{burst_id}.yaml')
+
+        runconfig_dict_out = runconfig_dict_in.copy()
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'input_file_group',
+                                 'burst_id'],
+                                [burst_id])
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'processing',
+                                 'geocoding',
+                                 'memory_mode'],
+                                'single_block')
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_dir'],
+                                dir_output)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'save_secondary_layers_as_hdf5'],
+                                 False)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_imagery_format'],
+                                 'GTiff')
+
+
+        if runconfig_dict_out['runconfig']['groups']['product_group']['save_mosaics']:
+            set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig', 'groups', 'product_group', 'save_mosaics'],
+                                False)
+            set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig', 'groups', 'processing', 'geocoding', 'save_nlooks'],
+                                True)
```

Comment:
This is fine, but please add something like "TODO fix this later" so we remember that the creation of `nlooks` should be fixed in the future (i.e., once mosaic doesn't need `nlooks` anymore).